contracts/core/src/xchain/OmniPortal.sol

```diff
@@ -366,17 +366,17 @@ contract OmniPortal is
 
         uint64 totalPower;
         XTypes.Validator memory val;
-        mapping(address => uint64) storage valSet = valSet[valSetId];
+        mapping(address => uint64) storage set = valSet[valSetId];
```

Comment:
Approving but don't love `set`, since that's an overloaded term, maybe consider underscores